src/services/codefixes/fixCannotFindModule.ts

```diff
@@ -4,9 +4,8 @@ namespace ts.codefix {
     const errorCodes = [Diagnostics.Could_not_find_a_declaration_file_for_module_0_1_implicitly_has_an_any_type.code];
     registerCodeFix({
         errorCodes,
-        getCodeActions: context => [
-            { fixId, ...tryGetCodeActionForInstallPackageTypes(context.host, context.sourceFile.fileName, getModuleName(context.sourceFile, context.span.start)) }
-        ],
+        getCodeActions: context => (codeAction => codeAction && [{ fixId, ...codeAction }])(

```

Comment:
Style: Use `const x = init; return f(x);` instead of `(x => f(x))(init)`. So in this case use `const codeAction = tryGet...; return codeAction && [{ fixId, ...codeAction }];